### PR TITLE
Changed size of plugs, color of text and box-shadow color

### PR DESCRIPTION
**Pull request by KhrystynaPavlikovska**
## Description

Changed size of plugs and images in the carousel.
Also, changed box-shadow color and color of text.
Bug: https://github.com/ita-social-projects/horondi_client_fe/issues/2209

#### Screenshots

|         Original          |         Updated          |
| :-----------------------: | :----------------------: |
|**image**|**image**|
|**image**|**image**|
|**image**|**image**|

### Checklist

- [x] 🔽 My branch is up-to-date with "development" branch
- [x] ✅All tests passed locally
- [x] ✨My changes working with up-to-date admin and back-end part locally, like charm
- [x] 🔗 Link pull request to issue

### CODE REVIEW

**Review by LubomyrKashuba — APPROVED** [review]

File: src/pages/product-details/product-images/product-images.styles.js [hunk 1]
```diff
@@ -103,7 +104,7 @@ export const useStyles = makeStyles((theme) => ({
   lastImage: {
     ...imgSides,
     filter: 'brightness(50%)',
-    boxShadow: theme.palette.carouselItem.shadow.boxShadow
+    boxShadow: 'inset 0 0 0 1000px rgba(0,0,0,.5)'
```
Comment (Pipuss) [inline src/pages/product-details/product-images/product-images.styles.js:107]: Isn't it better idea to create another instance of shadow in palette and move the value there?

**Review by Pipuss — CHANGES_REQUESTED** [review]

**Review by Pipuss — DISMISSED** [review]

File: src/pages/product-details/product-images/product-images.styles.js (hunk 1 above)
Comment (KhrystynaPavlikovska) [reply src/pages/product-details/product-images/product-images.styles.js]: fixed

**Review by Tolik170 — APPROVED** [review]

**Review by LubomyrKashuba — DISMISSED** [review]

**Review by Yevhendzyuba — APPROVED** [review]

File: src/components/app/app-theme/app.pallete.js [hunk 2]
```diff
@@ -13,6 +13,7 @@ const blue = '#1976D2';
 const red = '#F44336';
 const lightPing = '#FEEFEF';
 const green = '#4CAF50';
+const shadow = 'inset 0 0 0 1000px rgba(0,0,0,.5)';
```
Comment (maarekua) [inline src/components/app/app-theme/app.pallete.js:16]: Shadow is not applicable for current criteria.

**Review by maarekua — CHANGES_REQUESTED** [review]

File: src/components/app/app-theme/app.pallete.js (hunk 2 above)
Comment (KhrystynaPavlikovska) [reply src/components/app/app-theme/app.pallete.js]: fixed

**Review by Pipuss — DISMISSED** [review]

**Review by tarasSanotskyi — APPROVED** [review]

**Review by Yevhendzyuba — APPROVED** [review]

**Review by Tolik170 — DISMISSED** [review]

**Review by tarasSanotskyi — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

File: src/pages/product-details/product-images/product-images.styles.js [hunk 3]
```diff
@@ -117,15 +117,16 @@ export const useStyles = makeStyles((theme) => ({
...
-    padding: '0 5px',
-    color: theme.palette.black,
+
+    color: theme.palette.white,
     '@media (max-width: 500px)': {
       padding: '0',
       fontSize: '14px'
     }
   },
   imageItem: {
```
Comment (LubomyrKashuba) [inline src/pages/product-details/product-images/product-images.styles.js:127]: try to use ::backdrop

Comment (KhrystynaPavlikovska) [reply src/pages/product-details/product-images/product-images.styles.js:127]: ok) thnx

**Pull request merged**